setup.cfg

```diff
@@ -27,6 +27,8 @@ classifiers =
 install_requires =
     python-docx
     tinycss2
+tests_require =
+    pillow
```

Comment:
This is already handled by tox. Why do we need this? What command are you testing with?

Reply:
pytest

Reply:
It is also worth noting that `tests_require` is deprecated and not encouraged:

https://setuptools.readthedocs.io/en/latest/setuptools.html#new-and-changed-setup-keywords